packages/lesswrong/server/utils/httpUtil.ts

```diff
@@ -58,15 +58,29 @@ export function setCookieOnResponse({req, res, cookieName, cookieValue, maxAge}:
     // The server-side code depends on the cookie existing on the very first request, before the client can send back the cookie we set via header
     untypedReq.cookies = { [cookieName]: cookieValue };
   }
-  
-  (res as any).setHeader("Set-Cookie", `${cookieName}=${cookieValue}; Max-Age=${maxAge}`);
+
+  (res as any).setHeader("Set-Cookie", `${cookieName}=${cookieValue}; Max-Age=${maxAge}; Path=/`);
 }
 
 export function getAllCookiesFromReq(req: Request) {
   const untypedReq: any = req;
-  if (untypedReq.universalCookies)
-    return untypedReq.universalCookies
-  else
+
+  if (untypedReq.universalCookies) {
```

Comment:
This logic feels a bit confusing to me to read. I might have done something like

`if (untypedReq.cookies && untypedReq.universalCook) {
combine and return
},
if (untypedReq.cookies...`

Is there a reason you didn't do it this way?